.gitignore

```diff
@@ -16,4 +16,9 @@ log/
 *.pyc
 *~
 *.xls
-*.xlsx
\ No newline at end of file
+*.xlsx
+.idea
+*.iml
+*.ipr
+*.iws
+*.log
```

Comment:
These should be part of your personal global `.gitignore`, not checked in.

Reply:
We do have similar entries in our top level nuxeo/.gitignore ....

Reply:
Yeah but in my opinion they're wrong. Anything generated by personal tools (like IDEs and other editors) should be in a personal gitignore. Only extensions to ignore that are generated by build tools should be there (say, *.pyc, target, node_modules, ...). IMHO

Reply:
I agree. Just mentioned 'cause I was doing the same remark and didn't after looking up our .gitignore

Reply:
I copied this from the main Nuxeo Repository